programs/mmm/src/util.rs

```diff
@@ -783,6 +785,68 @@ pub fn check_allowlists_for_mint_ext(
     Err(MMMErrorCode::InvalidAllowLists.into())
 }
 
+pub fn check_allowlists_for_mpl_core<'info>(
+    allowlists: &[Allowlist],
+    asset: &Box<Account<'info, IndexableAsset>>,
+    allowlist_aux: Option<String>,
+) -> Result<()> {
+    if allowlists
+        .iter()
+        .any(|&val| val.kind == ALLOWLIST_KIND_METADATA)
```

Comment:
explore if we can make Core's onchain traits verifiable.

Reply:
will be address in the future when we support onchain attribute offer for mpl core